src/hyspecppt/hppt/hppt_view.py

```diff
@@ -503,27 +539,47 @@ def __init__(self, parent: Optional["QObject"] = None) -> None:
         self.Ei_edit = QLineEdit(self)
         self.Ei_label = QLabel("Incident energy &Ei:", self)
         self.Ei_label.setBuddy(self.Ei_edit)
+        tooltip_Ei = "Incident energy in meV, a number between 0 and 100."
+        self.Ei_edit.setToolTip(tooltip_Ei)
+        self.Ei_label.setToolTip(tooltip_Ei)
         self.Ei_validator = QDoubleValidator(bottom=0, top=100, parent=self)
         self.Ei_validator.setNotation(QDoubleValidator.StandardNotation)
         self.Ei_edit.setValidator(self.Ei_validator)
 
         self.Pangle_edit = QLineEdit(self)
         self.Pangle_label = QLabel("&Polarization angle:", self)
         self.Pangle_label.setBuddy(self.Pangle_edit)
+        tooltip_P_angle = "Polarization angle with respect to incident beam, in degrees, a number between -180 and 180."
+        self.Pangle_edit.setToolTip(tooltip_P_angle)
+        self.Pangle_label.setToolTip(tooltip_P_angle)
         self.Pangle_validator = QDoubleValidator(bottom=-180, top=180, parent=self)
         self.Pangle_validator.setNotation(QDoubleValidator.StandardNotation)
         self.Pangle_edit.setValidator(self.Pangle_validator)
 
         self.S2_edit = QLineEdit(self)
         self.S2_label = QLabel("Detector angle &S2:", self)
         self.S2_label.setBuddy(self.S2_edit)
+        tooltip_S2 = (
+            "Detector tank angle in degrees, a number between -100 and 100,"
+            + "\nbut not between -30 and 30. The detector coverage is +/- 30 degrees around S2"
+        )
+        self.S2_edit.setToolTip(tooltip_S2)
+        self.S2_label.setToolTip(tooltip_S2)
         self.S2_validator = AbsValidator(bottom=30, top=100, parent=self)
         self.S2_validator.setNotation(QDoubleValidator.StandardNotation)
         self.S2_edit.setValidator(self.S2_validator)
 
         self.Type_combobox = QComboBox(self)
         self.Type_label = QLabel("&Type:", self)
         self.Type_label.setBuddy(self.Type_combobox)
+        tooltip_type = (
+            "Quantity to plot. "
+            + PLOT_TYPES[0]
+            + " is the angle between"
```

Comment:
between should have a space after.
![image](https://github.com/user-attachments/assets/ec268ee4-d19c-4c7e-b059-86e5f723bcbb)